src/components/CardGrid.tsx

```diff
@@ -68,76 +104,54 @@ enum GridView {
    * 2x2 (4 cards visible at a time)
    */
   TWO_BY_TWO = '2x2',
+
+  /**
+   * 3x3 (6 cards visible at a time)
+   */
+  THREE_BY_THREE = '3x3',
+
   /**
    * 4x4 (16 cards visible at a time)
    */
   FOUR_BY_FOUR = '4x4',
   /**
    * 6x6 (36 cards visible at a time)
    */
-  SIX_BY_SIX = '6x6',
+  FIVE_BY_FIVE = '6x6',
```

Comment:
Should this be `SIX_BY_SIX` or `'5x5'`?

Reply:
dropped 6 x 6 doesn't work for our cards. also design has 4x4 for compact and 3x3 for expanded.